package.json

```diff
@@ -10,28 +10,36 @@
     "autoupdate-action": "bin/cli.js"
   },
   "scripts": {
+    "build": "tsc && ncc build bin/cli.ts --out dist",
```

Comment:
Do we need `tsc`? I thought `ncc` supported Typescript out of the box: https://github.com/vercel/ncc#with-typescript

Reply:
Also - could you try a docker build and see if the `Dockerfile` still works?

Reply:
oh cool, yeah I'll remove it. yes the dockerfile seems to work still